aspnet/fundamentals/configuration.rst

```diff
@@ -136,66 +136,74 @@ You configure options using the ``Configure<TOption>`` extension method. You can
 .. literalinclude:: configuration/sample/src/UsingOptions/Startup.cs
   :language: c#
   :linenos:
-  :lines: 24-40
+  :lines: 26-42
   :dedent: 8
   :emphasize-lines: 7,10-13
 
-When you bind options to configuration each property in your options type is bound to a configuration key of the form ``property:subproperty:...``. For example, the ``MyOptions.Option1`` property is bound to the key ``Option1``, which is read from the ``option1`` property in *appsettings.json*. Note that configuration keys are case insensitive.
+When you bind options to configuration, each property in your options type is bound to a configuration key of the form ``property:subproperty:...``. For example, the ``MyOptions.Option1`` property is bound to the key ``Option1``, which is read from the ``option1`` property in *appsettings.json*. Note that configuration keys are case insensitive.
 
-Each call to ``Configure<TOption>`` adds an ``IConfigureOptions<TOption>`` service to the service container that is used by the ``IOptions<TOption>`` service to provide the configured options to the application or framework. If you want to configure your options some other way (ex. reading settings from a data base) you can use the ``ConfigureOptions<TOptions>`` extension method to you specify a custom ``IConfigureOptions<TOption>`` service directly. 
+Each call to ``Configure<TOption>`` adds an ``IConfigureOptions<TOption>`` service to the service container that is used by the ``IOptions<TOption>`` service to provide the configured options to the application or framework. If you want to configure your options some other way (e.g. reading settings from a database) you can use the ``ConfigureOptions<TOptions>`` extension method to specify a custom ``IConfigureOptions<TOption>`` service directly. 
```

Comment:
I couldn't find any `ConfigureOptions` extension method. What should this sentence be replaced with?